tests/handlers/test_presence.py

```diff
@@ -508,15 +514,15 @@ def test_last_active(self) -> None:
 
 
 class PresenceHandlerTestCase(BaseMultiWorkerStreamTestCase):
-    def prepare(self, reactor, clock, hs):
+    def prepare(self, reactor: MemoryReactor, clock: Clock, hs: HomeServer) -> None:
         self.presence_handler = hs.get_presence_handler()
         self.clock = hs.get_clock()
 
     def test_external_process_timeout(self) -> None:
         """Test that if an external process doesn't update the records for a while
         we time out their syncing users presence.
         """
-        process_id = 1
+        process_id = "1"
```

Comment:
Ah,  this is some id string rather than a pid from the OS?

Was this a bug in the test?

Reply:
Yeah -- it seems to be called in real code via parsing a replication command, so it is all strings.